tekton/pipeline_run.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-with-version
```

Comment:
The name has to be unique! So the best idea would be to take something like `$(PR Number)-$(Short SHA)-$(short time)` or something like that.